ibis-server/docs/development.md

```diff
@@ -48,6 +45,13 @@ To run the tests:
 - Build the image: `just docker-build`
 - Run the container: `just docker-run`
 
+### Run the testing
+- Preapre the Wren Engine server (see [Wren Engine Example](../example/README.md)
+- Run pytest with `WREN_ENGINE_ENDPOINT` env. (The default port of Wren Engine is 8080)
+  - run specific data source test using [pytest marker](https://docs.pytest.org/en/stable/example/markers.html). There are some markers for different data sources. See the list in [pyproject.toml](https://github.com/Canner/wren-engine/blob/10d71be41ecb630e6d5f56834d2724afbf87710d/ibis-server/pyproject.toml#L50).
```

Comment:
I'm sorry late to notice this `pyproject.toml` link. How about using `../pyproject.toml`? If we use like with commit sha and line number, it will not follow the latest.